backend/slackbot/handler.js

```diff
@@ -112,33 +258,257 @@ async function getStatistics(applications){
+    }
+    slackAttachments[2].text = text;
+
+    let params = {
+      text: message,
+      attachments: slackAttachments
+    };
+
+    reports.push(params);
+  }
+
+  for (let report in reports) {
+    axios.post(hook, reports[report]);
+  }
 
+  return reports;
+}
+
+/**
+ * @param statistics list of statistics to output to slack
+ */
+async function getSentimentOverTime(statistics){
+  let reports = [];
+
+  // report represents the index in statistics list
   for (let report in statistics) {
-    let statsList = statistics[report].stats;
+    let message = '';
+    let attachments = [
+      {
+        'fallback': 'Sentiment Over Time',
+        'color': '#0066ff',
+        'title': 'Percentage of positive reviews by day',
+      }
+    ];
+
+    // Get the fields we want to use in our message to slack
+    let statsList = statistics[report];
+    let sentimentOverTime = statsList.sentimentOverTime;
+    let overallSentiment = statsList.overallSentiment;
+    let data = sentimentOverTime.data;
+    let labels = sentimentOverTime.labels;
+    let rawReviews = statsList.rawReviews;
+    let version = statsList.version;
+    let name = statsList.name;
+    let startDate = statsList.sentimentOverTime.labels[0];
+    let endDate = statsList.sentimentOverTime.labels[statsList.sentimentOverTime.labels.length-1];
+
+    // Determine the actual sentiment values as percentages
+    let sentiment = [Math.round(overallSentiment.POSITIVE), Math.round(overallSentiment.NEGATIVE), Math.round(overallSentiment.MIXED), Math.round(overallSentiment.NEUTRAL)];
+    let max = Math.max(sentiment[0], sentiment[1], sentiment[2], sentiment[3]);
 
-    for (let stat in statsList) {
-      message += statsList[stat].rawReviews;
-      message += '\n';
+    // Find the maximum sentiment value of(Pos, Neg, Neutral, Mixed)
+    let attitude = '';
+    if (max == sentiment[0]) {
+      attitude = 'Positive';
+    } else if (max == sentiment[1]) {
+      attitude = 'Negative';
+    } else if (max == sentiment[2]) {
+      attitude = 'Mixed';
+    } else {
+      attitude = 'Neutral';
     }
 
-    message += '\n\n';
+    // Build the slack message to send back with attachments
+    message += `Report for:\nApp: ${name}\nVersion: ${version}\n`;
+    message += `Between ${startDate} and ${endDate}, sentiment has been mostly ${attitude} for ${Object.keys(rawReviews).length} reviews`;
+
+    let text = '';
+
+    for (let i in data) {
+      let date =  labels[i];
+      let percent = Math.round(data[i]*100)/100;
+
+      if (data[i] != null) {
+        text += `On ${date}, ${percent}% of reviews were positive!\n`;
+      }
+    }
+
+    attachments[0].text = text;
+    let params = {
+      text: message,
+      attachments: attachments
+    };
+
+    reports.push(params);
+
+  }
+
+  for (let report in reports) {
+    axios.post(hook, reports[report]);
   }
 
+  return reports;
+}
+
+async function helpMessage(text='') {
+  let message = text + '\n';
```

Comment:
That's actually a good question, because they didn't ask for a help command to begin with I just added it in to give examples, so I can ask them if they have preference for help commands on Friday as well